modules/Emails/Email.php

```diff
@@ -4408,7 +4408,7 @@ public function sendOptInEmail(EmailAddress $emailAddress)
             return false;
         }
 
-        
+
         $confirmOptInTemplateId = $sugar_config['aop']['confirm_opt_in_template_id'];
```

Comment:
Please see same check here: https://github.com/salesagility/SuiteCRM/pull/4932/commits/8ed62120c9110b00004e253c8f3dafb5a9814342#diff-9aa62bd7609491e995053221eb0ed9fcR1102
If this check still needs, maybe we should outsource it to a common available function to avoid the code duplication.